lib/response-parsers/followed-profiles.js

```diff
@@ -21,82 +21,114 @@ const linkRelToNameMap = {
   related: 'related',
 };
 
-const parseLinks = nodes => _.reduce(nodes, (result, node) => {
-  const link = parseXMLNode(node, {
-    rel: 'string(@rel)',
-    type: 'string(@type)',
-    href: 'string(@href)',
-  }, xpath);
-
-  const { [link.rel || '']: name } = linkRelToNameMap;
-
-  /* beautify preserve:start */
-  return _.assign(result, { [name]: link });
-  /* beautify preserve:end */
-}, {});
-
-const parseCategories = nodes => _.reduce(nodes, (result, node) => {
-  const category = parseXMLNode(node, {
-    scheme: 'string(@scheme)',
-    value: 'string(@term)',
-  }, xpath);
-
-  const { [category.scheme || '']: name } = invertedCategorySchemes;
-
-  /* beautify preserve:start */
-  return _.assign(result, { [name]: category.value });
-  /* beautify preserve:end */
-}, {});
+const parseLinks = nodes =>
+  _.reduce(
+    nodes,
+    (result, node) => {
+      const link = parseXMLNode(
+        node,
+        {
+          rel: 'string(@rel)',
+          type: 'string(@type)',
+          href: 'string(@href)',
+        },
+        xpath
+      );
+
+      const { [link.rel || '']: name } = linkRelToNameMap;
+
+      /* beautify preserve:start */
+      return _.assign(result, { [name]: link });
+      /* beautify preserve:end */
+    },
+    {}
+  );
+
+const parseCategories = nodes =>
+  _.reduce(
+    nodes,
+    (result, node) => {
+      const category = parseXMLNode(
+        node,
+        {
+          scheme: 'string(@scheme)',
+          value: 'string(@term)',
+        },
+        xpath
+      );
+
+      const { [category.scheme || '']: name } = invertedCategorySchemes;
+
+      /* beautify preserve:start */
```

Comment:
these comments can be removed now since prettier-eslint doesn't work with jsbeautify

Reply:
done